nbs/tutorial.ipynb

```diff
@@ -4,7 +4,16 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Looks like there are a few of these to clean up.

Reply:
@jph00 , 
Since most users are using Linux, to avoid potential confusion on the docs.
I reruned all updated notebooks on Linux and push the changes to the repo.